advancedopenportal.xml

```diff
@@ -25,6 +25,8 @@
 	</update>
 	<languages>
                 <language tag="en-GB">language/en-GB/en-GB.com_advancedopenportal.ini</language>
+                <language tag="it-IT">language/it-IT/it-IT.com_advancedopenportal.ini</language>
+        
```

Comment:
This is not the proper way to create a language pack on Joomla. 
Each language needs to have its own xml file on a zip folder